### PR TITLE
Add Deliverance

### PR DESCRIPTION
**Pull request by Stumblinbear**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ A list of resources connected to the [SpaceTraders API](https://spacetraders.io/
 
 ## Client
 
+* [Deliverance](https://github.com/Stumblinbear/Deliverance) - VueJS-based web interface for Space Traders
```
Comment (jadlers) [inline README.md:16]: Please end the line with a period "." as per the the [contribution guidelines](https://github.com/SpaceTradersAPI/awesome-spacetraders/blob/main/contributing.md).

**Review by jadlers — CHANGES_REQUESTED** [review]
Congrats on submitting the first PR! :tada: 

Some changes needed before I can merge it:
1. A tiny modification needed in the readme addition.
2. You need to add instructions on how to get your project up and running in your repository. Preferably in its readme.

**Comment by artokun** [comment]
Can we nest this under Typescript too?

**Comment by jadlers** [comment]
@artokun at the moment I think it should be directly under client. The [contribution](https://github.com/SpaceTradersAPI/awesome-spacetraders/blob/main/contributing.md) guidelines says not to create a new subcategory before there is 3 items which belong under the category. This is to prevent too many subcategories from being made. Once there are a few TypeScript project we'll create the subcategory!

**Comment by Stumblinbear** [comment]
It doesn't use Typescript, so it would have to be under Javascript or VueJS.

**Review by jadlers — APPROVED** [review]

**Comment by jadlers** [comment]
Everything looks perfect! :ok_hand:

**Pull request merged**